src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -17,6 +17,11 @@ export class VeteranBenefitSummaryLetter extends React.Component {
   }
 
   handleChange(domEvent) {
+    window.dataLayer.push({
+      event: 'letter-benefit-option-clicked',
+      'letter-benefit-option': domEvent.target.id,
```

Comment:
While I'm thinking about it, i should also note that everything in GA resides in a single, global namespace. So, if you had a variable `checked` in two different places they'd each be overwriting each other constantly. Or more perniciously, if you set a `product` variable *only* on app startup it'd have like `gibct` and then use it in your event reporting and then th user opens `letters` in a different tab, overwriting `product`, all the future GIBCT events would get reported through tags labeled as `letters`. 

Which is why we try to always apply a meaningful prefix to our GA variables.